themes/default/content/docs/get-started/install/_index.md

```diff
@@ -129,6 +129,24 @@ Subsequent updates can be installed in the usual way:
 > choco upgrade pulumi
 ```
 
+### Winget
```

Comment:
Curious - should this be listed first?  I'm not sure of the ecosystem usage of Winget vs. Chocolatey, but my sense is that Winget is more "official" these days.  Is there any reason from our perspective why we would expect folks to have a better experience with Chocolatey?

Reply:
Relatedly, if that were the case, would it makes sense to update the Get Started tutorials to use Winget also?

Reply:
> Is there any reason from our perspective why we would expect folks to have a better experience with Chocolatey?

@lukehoban winget is indeed the "more official" package manager but my assumption is that Windows users are more familiar with chocolatey as it has been used and known for a much longer time than winget (which went [v1.0](https://devblogs.microsoft.com/commandline/windows-package-manager-1-0/) around 05/2021)

Another _catch_ of winget support is that pulumi releases are not immediately published to the winget registry: automated PRs such as [this one](https://github.com/microsoft/winget-pkgs/pull/56531) are sent to the official registry and have to be reviewed and merged by a maintainer so users might not always install latest pulumi because of this delay 

If ask me personally, I think the "Standalone Installer" could be the _default_ because it doesn't require any package manager (chocolatey, winget, etc.), it's just an MSI which means people can just uninstall it from control panel if they want and it is immediately created in [Releases of pulumi-winget](https://github.com/pulumi/pulumi-winget/releases) as soon as pulumi publishes a new version.

Reply:
> I think the "Standalone Installer" could be the default because it doesn't require any package manager

We don't have any auto-upgrade functionality, so in this sense it may be more beneficial to push users towards package managers than a manual installation. My 2c.

Reply:
> We don't have any auto-upgrade functionality

@mikhailshilkov That's a great point, these installers don't have that indeed.